1-exercises/J-string-substring/exercise2.js

```diff
@@ -14,11 +14,11 @@ let names = [
   "Arron Graham",
 ];
 
-names[0] = names[0].substring();
-names[1] = names[1].substring();
-names[2] = names[2].substring();
-names[3] = names[3].substring();
-names[4] = names[4].substring();
+names[0] = names[0].substring(0,6);
+names[1] = names[1].substring(0,7);
+names[2] = names[2].substring(0,4);
+names[3] = names[3].substring(0,4);
+names[4] = names[4].substring(0,5);
```

Comment:
you could also do e.g. `names[4].split(' ')[1]` to solve this exercise, but your solution is good